kolibri/content/api.py

```diff
@@ -44,10 +45,39 @@ def filter_recommendations_for(self, queryset, value):
         return data
 
     def filter_recommendations(self, queryset, value):
-        # return 25 random content nodes
-        pks = queryset.values_list('pk', flat=True).exclude(kind__in=['topic', ''])
-        count = min(pks.count(), 25)
-        return queryset.filter(pk__in=sample(list(pks), count))
+
+        from kolibri.logger.models import ContentSessionLog
+
+        if ContentSessionLog.objects.count() < 50:
```

Comment:
@rtibbles @jamalex what would be a good number of logs to have before we start making recommendations? Note: this would also go for other recommendation endpoints that we will have later.